src/Parameters/PHP81Implementation.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Stdlib\Parameters;
+
+use ArrayObject;
+use Laminas\Stdlib\ParametersInterface;
+
+use function http_build_query;
+use function parse_str;
+
+class PHP81Implementation extends ArrayObject implements ParametersInterface
```

Comment:
Should be `@internal` IMO: the implementation should not be part of the public API